### DISCUSSION TITLE
Bravo Standard Calendar + Non Fiscal Weeks

### DISCUSSION THREAD

**Discussion by Lewolze** [sql-bi/Bravo#602] [General]
Hello everyone,

I use the Standard calendar (Power BI) but in some cases I need to break down into weeks. I didn't manage to combine Bravo Standard Calendar with Weekly Calendar. At the same time I don't need fiscal weeks. Does anyone know how I can do it with Bravo Standard Calendar?

**Comment by marcosqlbi** [comment]
You should customize the template by adding the columns you want - start from https://docs.sqlbi.com/bravo/features/manage-dates/customize-date-template, follow links to related documentation and watch videos.
